### PR TITLE
Add committee affiliations

### LINKED ISSUE #61: add committee affiliations
At meetings/bordeaux2018/.

### PR DESCRIPTION
**Pull request by odow**
Close #61

### CODE REVIEW

File: meetings/bordeaux2018/index.md [hunk 1]
```diff
@@ -48,12 +48,12 @@ A preliminary schedule is as follows.
...
 ## Committee
 
-- Miles Lubin
-- Oscar Dowson
-- Benoît Legat
-- Joaquim Garcia
-- Juan Pablo Vielma
-- Issam Tahiri (Local Organization)
-- François Vanderbeck (Local Organization)
+- Miles Lubin (Google Research)
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: Just "Google"

**Comment by mlubin** [comment]
@issamt @blegat @joaquimg please confirm
We may want to avoid abbreviations except for "U."

File: meetings/bordeaux2018/index.md [hunk 2]
```diff
@@ -48,12 +48,12 @@ A preliminary schedule is as follows.
...
-- Miles Lubin
-- Oscar Dowson
-- Benoît Legat
-- Joaquim Garcia
-- Juan Pablo Vielma
-- Issam Tahiri (Local Organization)
-- François Vanderbeck (Local Organization)
+- Miles Lubin (Google)
+- Oscar Dowson (U. of Auckland)
+- Benoît Legat (UCL)
```
Comment (blegat) [inline meetings/bordeaux2018/index.md]: Could you change UCL to UCLouvain ? (UCL can be confused with University College de Londres)

**Comment by IssamT** [comment]
Yes, they are correct. Thanks.

**Pull request merged**